### ISSUE TITLE
Release of fasttext classifier from paper

### ISSUE THREAD

**Issue by akashgokul** [TristanThrush/perplexity-correlations#1]
Hi, Great work! Are there any plans to release the fasttext classifier used in the paper?

**Comment by TristanThrush** [comment]
Thanks for your interest - with our method, there isn't just one fasttext classifier. There is a different fasttext classifier trained for each target task, because our approach selects pretraining data targeting high performance on a particular benchmark or set of them. I'm happy to release all of the fasttext classifiers if you'd find them useful!

I'm planning to release the code to run the full experiments in our paper in an examples directory in this repo, stay tuned!

**Comment by TristanThrush** [comment]
(If you meant the datacomp-lm fasttext classifier that we use for comparison purposes, it is already released here: https://huggingface.co/mlfoundations/fasttext-oh-eli5)

**Comment by akashgokul** [comment]
Sorry for the confusion; yeah, I was wondering if you were going to release all of the fasttext classifiers. I was hoping to use them for my experiments. Thanks for the help!

**Comment by TristanThrush** [comment]
will ping you when i release them! Hopefully will be soon

**Comment by TristanThrush** [comment]
Hey @akashgokul here are the fasttext models: https://huggingface.co/perplexity-correlations
